src/components/ProfileList/profile-data.ts

```diff
@@ -28,38 +28,38 @@ export enum JobSpecialization {
   Fullstack = 'Fullstack',
 }
 
-export const profileData: ProfileListItems[] = [
-  {
-    id: 1,
-    name: 'Karolina Morwinska',
-    seniority: 'Senior',
-    jobSpecialization: JobSpecialization.Fullstack,
-    others: '+5 more',
-    country: 'Poland',
-    city: 'Warsaw',
-    remote: 'Remote',
-    technology: ['Javascript', 'React', 'Vue.js'],
-  },
-  {
-    id: 2,
-    name: 'Kristin Watson',
-    seniority: 'Junior',
-    jobSpecialization: JobSpecialization.Frontend,
-    others: '+5 more',
-    country: 'Poland',
-    city: 'Warsaw',
-    remote: 'Remote',
-    technology: ['Javascript', 'React', 'Vue.js'],
-  },
-  {
-    id: 3,
-    name: 'Veres Panna',
-    seniority: 'Mid',
-    jobSpecialization: JobSpecialization.Fullstack,
-    others: '+5 more',
-    country: 'Poland',
-    city: 'Warsaw',
-    remote: 'Remote',
-    technology: ['Javascript', 'React', 'Vue.js'],
-  },
-]
+// export const profileData: ProfileListItems[] = [
+//   {
+//     id: 1,
+//     name: 'Karolina Morwinska',
+//     seniority: 'Senior',
+//     jobSpecialization: JobSpecialization.Fullstack,
+//     others: '+5 more',
+//     country: 'Poland',
+//     city: 'Warsaw',
+//     remote: 'Remote',
+//     technology: ['Javascript', 'React', 'Vue.js'],
+//   },
+//   {
+//     id: 2,
+//     name: 'Kristin Watson',
+//     seniority: 'Junior',
+//     jobSpecialization: JobSpecialization.Frontend,
+//     others: '+5 more',
+//     country: 'Poland',
```

Comment:
**nit:** Why comment? If commenting not removing, let's add explanation in code why it's not removed

Reply:
deleted, it was a fragment used when we did not download profiles from the database, I guess.